test/clients/test_client.py

```diff
@@ -1,39 +1,35 @@
-from hypothesis import given
-from hypothesis.strategies import text
-
 from profiles.clients import Client
 
+name = 'journal'
+client_id = 'client_id'
+client_secret = 'client_secret'
+redirect_uris = ['example.com', 'example.org', 'testing.example.org']
 
-@given(text(min_size=1), text(min_size=1), text(min_size=1), text(min_size=1))
```

Comment:
I'm all in favor of property-based testing for domain data like the profiles themselves, while here it's exercising getters and setters so the maintenance overhead (updating the generation strategies) is high